calliope/analysis/plotting/energy_flows.py

```diff
@@ -0,0 +1,319 @@
+"""
```

Comment:
Again, can you change the filename to ``flows.py`` or ``flow.py`` (and function names accordingly)?